docs/setup-cluster/deploy-cluster/slurm/hpc-with-agent.rst

```diff
@@ -0,0 +1,153 @@
+.. _hpc-with-agent:
+
+####################
+ Agent on Slurm/PBS
+####################
+
+As an alternative to using the HPC Launcher, you may instead utilize the Determined agent. In this
+usage model, the system administrator creates a custom resource pool for each Determined user. You
+then start a Determined agent on one or more compute nodes of the cluster using Slurm or PBS
+commands to provide the resources for your resource pool. As work is submitted to this resource
+pool, it is distributed to the set of available agents. If your Slurm/PBS job is terminated (for
+example due to a time limit) before your Determined work is completed, your Determined work remains
```

Comment:
When you say `If your Slurm/PBS job is terminated`, you mean that the `agent` is terminated, which causes your Determined job to fail, right?

When you say `your Determined work remains in your resource pool`, you mean that the Determined experiment, for example, continues to be associated with the resource pool, but since there are not enough agents running for that resource pool (because they were terminated due to a timeout, for example), you must restart the agents again, correct?

Reply:
| When you say If your Slurm/PBS job is terminated, you mean that the agent is terminated, which causes your Determined job to fail, right?
right.  The only Slurm/PBS jobs in this model are the jobs running the agent.

|  (because they were terminated due to a timeout, for example), you must restart the agents again, correct?

Right.